src/styles/layout/index.md.njk

```diff
@@ -123,3 +123,20 @@ The desktop specific classes also allow you to set the width of the tablet break
 ### Nested grids
 
 {{ example({group: "styles", item: "layout", example: "nested", html: true, open: true, size: "m"}) }}
+
+### Display override classes
+
+Occasionally, you might need to override the default way that certain elements display on the page. You can use the display override classes for this.
```

Comment:
Non-blocking but something like the following wording would be more consistent with how we talk about the [width override classes](https://design-system.service.gov.uk/styles/spacing/#width-override-classes):

> If you need to override the default way that certain elements display on the page, you can use the display override classes.
>
> There are override classes for each of the following CSS `display` properties:

But @m-green might have some thoughts on this.